lib/VCMIDirs.cpp

```diff
@@ -23,7 +24,7 @@ namespace bfs = boost::filesystem;
 
 bfs::path IVCMIDirs::userLogsPath() const { return userCachePath(); }
 
-bfs::path IVCMIDirs::userSavePath() const { return userDataPath() / "Saves"; }
+bfs::path IVCMIDirs::userSavePath() const { return userDataPath() / "saves"; }
```

Comment:
that's a dangerous change for case-sensitive filesystems (e.g. default Linux choice)

Reply:
I think I was saw somewhere it doesn't matter and all folder are processed as case-insensive. This change exatly doesn't matter and can be reverted.

Reply:
Virtual filesystem that VCMI uses is indeed case-insensitive, but VCMIDirs is low-level class that is used to initialize said virtual filesystem, and probably the only class where case-sensitivity does matter.

So yeah, should be reverted to be safe.

Reply:
Will revert it just to be sure to not break anything.